docs/model-dev-guide/api-guides/batch-processing/batch-process-api-ug.rst

```diff
@@ -37,6 +37,17 @@ This API automatically handles the following for you:
 This is a flexible API that can be used for many different tasks, including batch (offline)
 inference.
 
+If you have some trained models in a :class:`~determined.experimental.checkpoint.Checkpoint` or a
+:class:`~determined.experimental.model.Model` with multiple
+:class:`~determined.experimental.model.ModelVersion` inside, you can associate the Trial with the
+:class:`~determined.experimental.checkpoint.Checkpoint` or
+:class:`~determined.experimental.model.ModelVersion` used in a given inference run to aggregate
+custom inference metrics.
+
+We can then query those :class:`~determined.experimental.checkpoint.Checkpoint` or
+:class:`~determined.experimental.model.ModelVersion` objects using the Python SDK to see all metrics
+associated with them.
+
```

Comment:
[edits: fixed the grammar, made the voice consistent, and added a link]

If you have some trained models in a :class:`~determined.experimental.checkpoint.Checkpoint` or a
:class:`~determined.experimental.model.Model` with more than one
:class:`~determined.experimental.model.ModelVersion` inside, you can associate the Trial with the
:class:`~determined.experimental.checkpoint.Checkpoint` or
:class:`~determined.experimental.model.ModelVersion` used in a given inference run to aggregate
custom inference metrics.

You can then query those :class:`~determined.experimental.checkpoint.Checkpoint` or
:class:`~determined.experimental.model.ModelVersion` objects using the :ref:`Python SDK <python-sdk>` to see all metrics
associated with them.